facility-manager.html

```diff
@@ -0,0 +1,806 @@
+menu-id: l1-workflow2
+pdfs:
+    - facility-manager.pdf
+description: Descriptions of common workflows for Facility Managers and links to useful sections of User Help and Developer documentation.
+version:
+    - yes
+---
+<div class="page-navigation">
+<h2>Contents</h2>
+<p>
+<a href="#introduction">Introduction</a><br />
+<a href="#using">Using OMERO</a><br />
+<a href="#permissions">Permissions on OMERO</a><br />
+<a href="#access">Access control, authentication, user accounts</a><br />
+<a href="#manage">Managing Groups</a><br />
+<a href="#import">Importing</a><br />
+<a href="#data">Data structure</a><br />
+<a href="#measure">Measurement and analysis</a><br />
+<a href="#disk">Disk usage</a><br />
+<a href="#email">Emailing users</a><br />
+<a href="#upgrade">Upgrading</a><br />
+<a href="#publish">Using OMERO for publication</a><br />
+<a href="#training">Getting users started</a>
+</p>
+</div><!-- End Page Navigation -->
+
+<div class="line-block">
+<div class="line"><br /></div>
+</div>
+
+<h2><a class="anchor" id="introduction"></a>Introduction</h2>
+
+<p>This section brings together descriptions of common workflows for Facility Managers and links to useful sections of User Help and technical documentation.</p>
+
+<h2><a class="anchor" id="install"></a>Installation and configuration</h2>
+
+<p>Documentation covering the installation and configuration of the OMERO server is in the <a href="http://www.openmicroscopy.org/site/support/omero5.2/sysadmins/index.html" target="_blank" class="external-url">System Administrator</a> documentation on the main OME website.</p>
+
+<h2>Specifications for hardware</h2>
+
+<p>This is a frequently asked question, and the simplistic (if slightly unhelpful) answer is the best you can manage.</p>
+
+<p>Apart from basic CPU number, speed and physical disk size, there are a number of other issues to be taken into account, including the nature of the storage being used for the data, the connection type, and the speed of the network used to access the server.</p>
+
+<p>Examples of system specifications and set-ups for production OMERO servers have kindly been provided by a number of institutes to serve as illustrations of working systems. These are shown in the <a href="http://www.openmicroscopy.org/site/support/omero5.2-staging/sysadmins/server-setup-examples.html" target="_blank" class="external-url">Example production server set-ups</a> page on the OME website.</p>
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h2><a class="anchor" id="using"></a>Using OMERO</h2>
+
+<p>Details on downloading the OMERO clients and accessing the OMERO server are in <a href="http://help.openmicroscopy.org/getting-started-5.html" target="_blank" class="external-url">Getting Started with OMERO</a> in User Help.</p>
```

Comment:
This is now OMERO.insight in the menu, the different title here is a bit confusing